### PR TITLE
[HB-5985]: ironSource Bidding support.

### PR DESCRIPTION
**Pull request by cb-jpadilla**
- Added bidding support for the ironSource adapter.
- Updated adapter version to 4.7.7.0.0.1

### CODE REVIEW

File: IronSourceAdapter/src/main/java/com/chartboost/mediation/ironsourceadapter/IronSourceAdapter.kt [hunk 1]
```diff
@@ -211,8 +211,11 @@ class IronSourceAdapter : PartnerAdapter {
         request: PreBidRequest,
     ): Map<String, String> {
         PartnerLogController.log(BIDDER_INFO_FETCH_STARTED)
-        PartnerLogController.log(BIDDER_INFO_FETCH_SUCCEEDED)
-        return emptyMap()
+
+        val token = IronSource.getISDemandOnlyBiddingData(context) ?: ""
+
+        PartnerLogController.log(if (token.isNotEmpty()) BIDDER_INFO_FETCH_SUCCEEDED else BIDDER_INFO_FETCH_FAILED)
+        return mapOf("token" to token)
```
[the remark this answers is not in the document]
Comment (cb-jpadilla) [reply IronSourceAdapter/src/main/java/com/chartboost/mediation/ironsourceadapter/IronSourceAdapter.kt:218]: The ironSource SDK is currently returning us with a null, which we can handle and not include in our request. We are not fully sure if they'd want to have empty string yet.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.7.7.0.0.1
+- Added bidding support.
```
[the remark this answers is not in the document]
Comment (cb-jpadilla) [reply CHANGELOG.md:7]: Ah, good call!

File: IronSourceAdapter/src/main/java/com/chartboost/mediation/ironsourceadapter/IronSourceAdapter.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (bichenwang) [reply IronSourceAdapter/src/main/java/com/chartboost/mediation/ironsourceadapter/IronSourceAdapter.kt:218]: you sure you handle and not include in the request? If ironsource returns null, you ?: it into "" which then you return as a map of "token" to "", no?

**Comment by bwised** [comment]
Superseded by https://github.com/ChartBoost/chartboost-mediation-android-adapter-ironsource/pull/68

**Pull request closed without merging**